openmls/src/framing/plaintext.rs

```diff
@@ -342,12 +375,12 @@ impl TryFrom<u8> for ContentType {
     }
 }
 
-impl From<&MlsPlaintextContentType> for ContentType {
-    fn from(value: &MlsPlaintextContentType) -> Self {
+impl From<&MlsContentBody> for ContentType {
+    fn from(value: &MlsContentBody) -> Self {
         match value {
```

Comment:
Isn't this `value.content_type()`? Or the other way round, but it's the same match.

Reply:
Yeah. I vote to just delete this conversion. Generally, I'm a fan of using `{,Try}{From,Into}` but in this particular case it seems a bit off? What do you think? It is not used in the codebase and -- when I see correctly -- I havn't touched `MlsContentBody::into()` in this PR which means it wasn't used before.

Reply:
Hm... can't tell for sure what feels so off about it really. So we can also keep it and I'll just simplify the match.